embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/anr/sigquit/SigquitDataSource.kt

```diff
@@ -36,18 +34,12 @@ internal class SigquitDataSource(
 
     fun saveGoogleAnr(timestamp: Long) {
```

Comment:
should we call this sigquit too?

Reply:
I'll rename it.

Sigquit is the C signal that the Android OS sends when an ANR happens. 'Google ANR' is just internal jargon that we've created - IMO it'd be better to be precise with our meaning & use sigquit terminology instead when discussing this.